src/DbLoader.php

```diff
@@ -306,6 +306,6 @@ public function getIncrementalLastValue(): string
             ->whereNotNull($this->updateColumn)
             ->orderByDesc($this->updateColumn)
             ->limit(1)
-            ->value($this->updateColumn);
+            ->value($this->updateColumn) ?? '';
```

Comment:
Would it make more sense to return `null|string` ?

Reply:
Absolutely. I considered that, but hoped for this to cause the least amount of disruption. The [if ($incremental) ](https://github.com/ziffmedia/laravel-etls/blob/master/src/DbExtractor.php#L74) check correctly fails on an empty string, so everything else downstream runs correctly. 

We could change the return type, but that would require additional refactors elsewhere and a version bump for anyone who might be using it (which is only us based on the repos I have access to, but I don't know what else is out there).